### DISCUSSION TITLE
Pylint unable to import errors

### DISCUSSION THREAD

**Discussion by axieax** [mason-org/mason.nvim#709] [Q&A]
Hello, I'm using this mason to install pylint, but am having issues with pylint being unable to import from my pip installed packages. From looking at the source code, this is due to the pip3 manager installing packages into a virtual environment, which is separate from my local python3 installation and the packages there. Is there a way to resolve this issue?

**Comment by getlost01** [comment]
It sounds like you're having an issue with Pylint being unable to import packages that you've installed with `pip `because they are installed in a virtual environment, which is separate from your local Python installation. There are a few ways you can try to resolve this issue:

1. Activate the virtual environment where the packages are installed: If you're using `virtualenv or venv `to create and manage your virtual environments, you can activate the virtual environment where the packages are installed by using the `source` command followed by the path to the `activate` script. For example:
`source path/to/venv/bin/activate`

2. Run Pylint within the virtual environment: If you don't want to activate the virtual environment, you can run Pylint within the virtual environment by using the` python` executable from the virtual environment. For example:
`path/to/venv/bin/python -m pylint my_module.py`

3. Install the packages globally: If you want Pylint to be able to import the packages from your local Python installation, you can install them globally by using the `--user` flag with `pip`. For example:
`pip install --user package_name`

I hope one of these options helps resolve the issue you're having with Pylint not being able to import your packages.
